ccmlib/dse_node.py

```diff
@@ -192,8 +192,8 @@ def dse(self, dse_options=None):
         dse = common.join_bin(self.get_install_dir(), 'bin', 'dse')
         args = [dse]
         args += dse_options
-        p = subprocess.Popen(args, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        return handle_external_tool_process(p, args)
+        p = subprocess.Popen(args, env=env)  # Spawn an interactive shell, cannot pipe stdout/std err
```

Comment:
Ah, just saw the bug report for this. So, I know that `spark` is an interactive command -- are all the others?

The issue I see here is that `handle_external_tool_process` will raise an exception if the `returncode` is nonzero, which I think is the behavior we want for most of these commands. Maybe you want to make a change to the handler method that changes in/out behavior when input/output isn't redirected?